dist.sh

```diff
@@ -45,9 +45,10 @@ done
 
 rnd=$(LC_ALL=C tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c10)
 docker buildx create --use --name nsq-$rnd
-docker buildx build --tag nsqio/nsq:v$version . --platform linux/amd64,linux/arm64 --push
+docker buildx build --tag nsqio/nsq:v$version --platform linux/amd64,linux/arm64 .
 if [[ ! $version == *"-"* ]]; then
-    echo "Tagging nsqio/nsq:v$version as the latest release."
-    docker buildx build --tag nsqio/nsq:latest . --platform linux/amd64,linux/arm64 --push
+    echo "Tagging nsqio/nsq:v$version as the latest release"
+    shas=$(docker manifest inspect nsqio/nsq:$version |\
+        grep digest | awk '{print $2}' | sed 's/[",]//g' | sed 's/^/nsqio\/nsq@/')
+    docker manifest create nsqio/nsq:latest $shas
 fi
-docker buildx rm nsq-$rnd
```

Comment:
i think without the `buildx rm` the builders will pile up.  not sure how much resources they use.  

it might be possible to do a builder cleanup prior to setting up new builders.  for now on our systems if something happens and the builder gets orphan'd, i've just been removing it manually.

Reply:
I think one could also skip the `create` and `use` steps, and just `buildx build` for local builds, right? It would just cause build cache to accumulate until you `docker system prune` or similar?

Reply:
I like having a namespace to inspect as an artifact of the build. I don't expect this script to be used in any automated context, so practically speaking it's fine.

Reply:
for local builds i think just doing an old school `docker build` is fine.  it's really only for the multi-arch push that we need to do the create/use/build-and-push chain.